geometry_msgs/Point[] world_coordinates
geometry_msgs/Point[] map_coordinates
